en/docs/reusable-components/choreo-connectors-list.md

```diff
@@ -9,31 +9,33 @@ The following lists the connections you can use to interact with generic protoco
-| **Azure Cosmos DB** | Integrates with Azure Cosmos DB to manage documents and stored procedures. You can perform operations such as creating documents, executing stored procedures, etc. |
-| **MongoDB**         | Integrates with MongoDB to manage databases, collections, and documents. You can perform operations such as getting databases, inserting documents, etc. |
-| **Redis**           | Integrates with Redis to manage different data structures. You can perform a range of operations depending on the data structure. |
-| **AWS S3**          | Integrates with AWS S3 to manage cloud storages. You can perform operations such as creating, listing, and deleting buckets as well as objects. |
-| **AWS SQS**         | Integrates with AWS SQS to manage messages and queues. You can perform operations such as creating queues, receiving messages, sending messages, deleting messages, and deleting queues. |
-| **People API**      | Integrates with the People API to manage your contacts. You can perform operations such as getting, listing, creating, deleting, and updating contacts. |
-| **Gmail**           | Integrates with Gmail to perform operations such as sending an email, receiving an email, etc.  |
-| **Google Calendar** | Integrates with Google Calendar to perform operations such as creating an event, deleting an event, etc. |
-| **Google Sheets**   | Integrates with Google Sheets to perform operations such as reading data, formatting data, etc. |
-| **Google Drive**    | Integrates with Google Drive to perform operations such as creating files and folders, getting items saved in the drive (e.g., retrieving a file), renaming files, deleting files, etc. |
-| **Zoom**            | Integrates with Zoom to schedule and manage all your meeting information. You can perform operations such as creating, updating, listing, and deleting meetings. |
-| **Spotify**         | Integrates with Spotify to manage your favorite playlists. You can perform operations such as creating, updating, and retrieving playlists. |
-| **SendGrid**        | Integrates with SendGrid to deliver transactional and marketing emails. You can perform operations such as sending emails, working with alerts, and working with subusers. |
-| **Medium**          | Integrates with Medium to access online publications. You can perform operations such as creating user posts, retrieving user details, and retrieving user publications. |
-| **LeanIX**          | Integrates with the LeanIX integration API to perform operations to manage your SaaS solutions.  |
+| **Connection**       | **Description**                                                                                |
+|----------------------|------------------------------------------------------------------------------------------------|
+| **Mail by Choreo**   | Sends email via Choreo to a specific address with a specific subject and a message body.        |
```

Comment:
Should we change "email" to "emails" or "an email"?

Reply:
"an email" seems better. 
Updated.